components/movementsensor/mpu6050/mpu6050.go

```diff
@@ -283,6 +283,10 @@ func (mpu *mpu6050) LinearVelocity(ctx context.Context, extra map[string]interfa
 	return r3.Vector{}, movementsensor.ErrMethodUnimplementedLinearVelocity
 }
 
+func (mpu *mpu6050) LinearAcceleration(ctx context.Context, exta map[string]interface{}) (r3.Vector, error) {
+	return r3.Vector{}, movementsensor.ErrMethodUnimplementedLinearAcceleration
```

Comment:
Like the ADXL345, I kinda want to `return mpu.linearAcceleration, nil` instead. and again, that will require locking the mutex, and checking (and possibly clearing) `mpu.lastError`.